pathways/simulation.py

```diff
@@ -165,17 +218,41 @@ def run_simulation(
             missing=0,
             num_inspections=0,
             num_boxes=0,
-            num_boxes_inspected=0,
+            num_stems=0,
+            avg_boxes_opened_completion=0,
+            avg_boxes_opened_detection=0,
+            pct_boxes_opened_completion=0,
+            pct_boxes_opened_detection=0,
+            avg_stems_inspected_completion=0,
+            avg_stems_inspected_detection=0,
+            pct_stems_inspected_completion=0,
+            pct_stems_inspected_detection=0,
+            pct_sample_if_to_detection=0,
+            pct_pest_unreported_if_detection=0,
             true_infestation_rate=0,
+            missed_infestation_rate=0,
+            intercepted_infestation_rate=0,
         )
     except AttributeError:
         # Python 2 fallback
         totals = lambda: None  # noqa: E731
-        totals.missing = 0
-        totals.num_inspections = 0
-        totals.num_boxes = 0
-        totals.num_boxes_inspected = 0
-        totals.true_infestation_rate = 0
+        totals.missing=0,
+        totals.num_inspections=0,
+        totals.num_boxes=0,
+        totals.num_stems=0,
+        totals.avg_boxes_opened_completion=0,
+        totals.avg_boxes_opened_detection=0,
+        totals.pct_boxes_opened_completion=0,
+        totals.pct_boxes_opened_detection=0,
+        totals.avg_stems_inspected_completion=0,
+        totals.avg_stems_inspected_detection=0,
+        totals.pct_stems_inspected_completion=0,
+        totals.pct_stems_inspected_detection=0,
+        totals.pct_sample_if_to_detection=0,
+        totals.pct_pest_unreported_if_detection=0,
+        totals.true_infestation_rate=0,
+        totals.missed_infestation_rate=0,
+        totals.intercepted_infestation_rate=0,
```

Comment:
Unlike the other places, here should be no commas at the end of the line.

Reply:
Removed commas.